pkg/utils/constants.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2018 The Kubernetes Authors.
```

Comment:
Kubernetes authors? How? Why? This seems to be code specific to oVirt....

Reply:
Done